coral/views/file_template.py

```diff
@@ -277,107 +282,115 @@ def insert_custom(self, document, k, v, config=None):
 
-    ACTIVITY_LOCALITIES_ADMIN_AREA_NODEGROUP = 'a5416b46-f121-11eb-8f2d-a87eeabdefba'
-    ACTIVITY_AREA_NAME_NODE = 'a5416b53-f121-11eb-a507-a87eeabdefba'
+#     ACTIVITY_SYSTEM_REF_NODEGROUP = "e7d695ff-9939-11ea-8fff-f875a44e0e11"
+#     ACTIVITY_SYSTEM_REF_RESOURCE_ID_NODE = "e7d69603-9939-11ea-9e7f-f875a44e0e11"
 
-    ACTIVITY_ADDRESSES_NODEGROUP = 'a5416b3d-f121-11eb-85b4-a87eeabdefba'
-    ACTIVITY_COUNTY_NODE = 'a541e034-f121-11eb-8803-a87eeabdefba'
+#     ACTIVITY_LOCALITIES_ADMIN_AREA_NODEGROUP = 'a5416b46-f121-11eb-8f2d-a87eeabdefba'
+#     ACTIVITY_AREA_NAME_NODE = 'a5416b53-f121-11eb-a507-a87eeabdefba'
 
-    ACTIVITY_NAME_NODEGROUP = '4a7bba1d-9938-11ea-86aa-f875a44e0e11'
-    ACTIVITY_NAME_NODE = '4a7be135-9938-11ea-b0e2-f875a44e0e11'
-
-    def __init__(self, resource_instance):
-        self.resource_instance = resource_instance
-        self.datatype_factory = DataTypeFactory()
-        self.tiles = resource_instance.tiles
-        self.mapping = {
-            "Licensee": "",
-            "Townland": "",
-            "County": "",
-            "Site Name": "",
-            "Duration": "",
-            "Duration Dates": "",
-            "License Number": "",
-        }
-        self.activity_resource = None
-        self.activity_tiles = None
+#     ACTIVITY_ADDRESSES_NODEGROUP = 'a5416b3d-f121-11eb-85b4-a87eeabdefba'
+#     ACTIVITY_COUNTY_NODE = 'a541e034-f121-11eb-8803-a87eeabdefba'
 
-    def get_value_from_tile(self, tile, node_id):
-        current_node = models.Node.objects.get(nodeid=node_id)
-        datatype = self.datatype_factory.get_instance(current_node.datatype)
-        returnvalue = datatype.get_display_value(tile, current_node)
-        return "" if returnvalue is None else returnvalue
+#     ACTIVITY_NAME_NODEGROUP = '4a7bba1d-9938-11ea-86aa-f875a44e0e11'
+#     ACTIVITY_NAME_NODE = '4a7be135-9938-11ea-b0e2-f875a44e0e11'
 
-    def get_activity_resource(self):
-        license_system_ref_tile = Tile.objects.filter(
-            resourceinstance_id=self.resource_instance.resourceinstanceid,
-            nodegroup_id=self.LICENSE_SYSTEM_REF_NODEGROUP
-        ).first()
-        ref_num = (
-            license_system_ref_tile.data
-            .get(self.LICENSE_SYSTEM_REF_RESOURCE_ID_NODE)
-            .get("en")
-            .get("value")
-        )
-        activity_system_ref_query = {
-            f"data__{self.ACTIVITY_SYSTEM_REF_RESOURCE_ID_NODE}__en__value__icontains": ref_num,
-        }
-        activity_system_ref_tile = Tile.objects.filter(**activity_system_ref_query).exclude(
-            nodegroup_id=self.ACTIVITY_SYSTEM_REF_NODEGROUP,
-            resourceinstance_id=self.resource_instance.resourceinstanceid,
-        ).first()
-        activity_resource = Resource.objects.filter(
-            pk=activity_system_ref_tile.resourceinstance.resourceinstanceid
-        ).first()
-        return activity_resource
+#     def __init__(self, resource_instance):
```

Comment:
Same here for removing commented code if its not being used

Reply:
removed